daemon/daemon.go

```diff
@@ -779,7 +779,7 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 	debug.RegisterStatusObject("ipam", d.ipam)
 
```

Comment:
Commit misses sign-off

Reply:
Is this a comment for me?

Reply:
@ryanmcnamara yes. Please squash all commits into the first one and then you can sign-off the resulting commit with `git commit --amend --sign-off`

Reply:
Thanks for the explanation, done

Reply:
This is resolved now that all commits are rebased into one commit with a sign-off.